### DISCUSSION TITLE
Speaker internal

### DISCUSSION THREAD

**Discussion by tuFFo126** [UzixLS/zx-sizif-512#33] [PCB Build]
Internal speaker goldpin. What parameters should the speakers have for the sound to be heard?
When I connected the speaker from the PC, there was practically nothing to hear, even when the internal potentiometer was set to the max.

**Comment by UzixLS** [comment]
You need amp like this http://www.drunknretro.com/2020/11/zx-spectrum-next-internal-speaker-mini-amplifier-mod.html
